src/Link.ts

```diff
@@ -40,7 +56,7 @@ export class Link extends WidgetBase<LinkProperties> {
 
 	protected render(): DNode {
 		const { to } = this.properties;
-		const props = { ...this.properties, to: undefined, isOutlet: undefined, params: undefined, routerKey: undefined };
+		const props = { ...this.properties, onclick: this._onClick, onClick: undefined, to: undefined, isOutlet: undefined, params: undefined, routerKey: undefined, router: undefined };
```

Comment:
It is fine for now.  Let me think on it, because it would be nice for it to work with the typings too...  If we could model a key deleter...